python/sglang/utils.py

```diff
@@ -2,12 +2,19 @@
 
 import base64
 import json
+import os
 import threading
 import urllib.request
+from concurrent.futures import ThreadPoolExecutor
 from io import BytesIO
 from json import dumps
 
+import cv2
+import numpy as np
 import requests
+from PIL import Image
```

Comment:
move 
`import cv2` and `from PIL import Image` under the function `encode_frame` so we can make them optional depenendcy